### PR TITLE
Fix examples

### PR DESCRIPTION
**Pull request by scouturier**
Updating examples parameters to "hostname" to reflect documentation.

### CODE REVIEW

File: deploy/examples/005_advanced_config_example_cr.yaml [hunk 1]
```diff
@@ -16,7 +16,7 @@ spec:
     deploymentStrategy: RollingUpdate
     skipSetup: false
     externalAccess:
-      configHostname: example-quayecosystem-quay-enterprise.apps
+      hostname: example-quayecosystem-quay-enterprise.apps
```
Comment (sabre1041) [inline deploy/examples/005_advanced_config_example_cr.yaml]: Does this require a change? `configHostname` is correct for specifying the location for the Config Route

**Review by sabre1041 — COMMENTED** [review]
@scouturier Thanks for helping update some of the examples. Added one comment

File: deploy/examples/005_advanced_config_example_cr.yaml (hunk 1 above)
Comment (scouturier) [reply deploy/examples/005_advanced_config_example_cr.yaml]: You are right, I will fix that and resubmit!

**Review by sabre1041 — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**